generated/nidaqmx/_lib.py

```diff
@@ -89,6 +90,10 @@ def __init__(self, library):
         self._library = library
         self._lib_lock = threading.Lock()
 
+    @property
+    def library(self):
+        return str(self._library).split(' ')[1].strip("',")
```

Comment:
I think you should simplify/optimize this. This work is duplicated by most API calls.

`DaqLibImporter._import_lib` can save the encoding at the time that the library is loaded.
- It could save the encoding as `self._encoding` in the `DaqLibImporter` class. Or as `self.encoding`, and then you don't need to write a `@property` method.
- Or it could save the encoding in the `DaqFunctionImporter` by passing it as another constructor parameter. However, I see no reason why we would mix and match DLLs. Both `DaqFunctionImporter` objects are for the same DLL; they just have different calling conventions.